doc/api/modules.md

```diff
@@ -481,81 +481,122 @@ added: v0.1.16
 
 * {Object}
 
-The `module.exports` object is created by the Module system. Sometimes this is
-not acceptable; many want their module to be an instance of some class. To do
-this, assign the desired export object to `module.exports`. Note that assigning
-the desired object to `exports` will simply rebind the local `exports` variable,
-which is probably not what you want to do.
+`module.exports` object is used to export values in a js file. It is
```

Comment:
* I don't understand why `The` was removed as the first word of this sentence.
* Do not use `js` as an abbreviation for `JavaScript`.